lib/cisco_node_utils/bgp_af.rb

```diff
@@ -244,7 +243,11 @@ def default_advertise_l2vpn_evpn
 
     # dampen_igp_metric
     def dampen_igp_metric
-      config_get('bgp_af', 'dampen_igp_metric', @get_args)
+      match = config_get('bgp_af', 'dampen_igp_metric', @get_args)
```

Comment:
I mean n7k does not show dampen_igp_metric 60 in show run bgp all when nothing configured.
So when nothing we need to return default. And we need to match no dampen_igp_metric. And return nil.